OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -255,6 +273,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="left">25</dim>
                                         <dim type="intercolumn">11</dim>
                                     </dimensions>
+                                    <note> The characters per line are approximately 7-8. The exact number is difficult to ascertain due to erased ink.</note>
```

Comment:
Has the ink been intentionally erased? Because I think "erasure" implies an intentional action upon the object

Reply:
It is not clear, perhaps not intentionally. I've replaced the sentence with "due to faded or erased ink."